opal/corelib/helpers.rb

```diff
@@ -11,18 +11,22 @@ def self.type_error(object, type, method = nil, coerced = nil)
     end
   end
 
-  def self.coerce_to(object, type, method)
+  def self.coerce_to(object, type, method, argument = undefined)
```

Comment:
Although there's probably no real case for this I think at this point this one should be `args` and used as `object.__send__ method, *args`

like in:

```
obj.to_enum(method = :each, *args) → enum
obj.to_enum(method = :each, *args) {|*args| block} → enum
```

Reply:
It's right! Updated as https://github.com/opal/opal/pull/1986/commits/dcdfbb28503c6bad6b341c968abd4bbe725dac66